projects/dsp-ui/src/lib/viewer/values/link-value/link-value.component.html

```diff
@@ -1,5 +1,5 @@
 <div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
-  <p class="rm-value">{{valueFormControl.value?.label}}</p>
+  <p class="rm-value" (click)="refResClicked()">{{valueFormControl.value?.label}}</p>
```

Comment:
Could this new thing be made visible in the playground?

Reply:
maybe there could be a custom page for this where the `@Output` would be registered

Reply:
If I'm understanding the question correctly, I don't think this would be possible since `@Output` event emissions can only be listened to by the direct parent, which is why I had to make that whole `ValueOperationEvent` service.

Reply:
Moved to a new task https://dasch.myjetbrains.com/youtrack/issue/DSP-578